lte/gateway/python/magma/subscriberdb/crypto/milenage.py

```diff
@@ -12,12 +12,19 @@
 """
 
 import hmac
-
 from Crypto.Cipher import AES
 from Crypto.Random import random
-
 from .lte import BaseLTEAuthAlgo
+from typing import NamedTuple
 
+FiveGRanAuthVector = NamedTuple(
+    'FiveGRanAuthVector', [
+        ('rand', bytes),
+        ('xres_star', bytes),
+        ('autn', bytes),
+        ('kseaf', bytes),
+    ],
+)
```

Comment:
why don't you import this from .lte?

Reply:
+1 no need to define this in multiple places

Reply:
Comment Addressed.